src/components/Contributors.tsx

```diff
@@ -38,6 +37,7 @@ const Contributors: React.FC<IProps> = () => {
       <Flex flexWrap="wrap">
         {contributorsList.map((contributor, idx) => (
           <LinkBox
+            key={contributor.name + idx}
```

Comment:
`contributor.name` is should be enough as a key here

Reply:
Was concerned that `contributor.name` may not actually be unique for all items... just switched it to use `contributor.login` instead which I believe should always be unique